apps/codeforafrica/src/lib/api.ghost/index.js

```diff
@@ -0,0 +1,47 @@
+import { getAllPosts, getPost } from "@/codeforafrica/lib/api.ghost/posts";
+
+export async function getAllOpportunities() {
+  const allPosts = await getAllPosts();
+  const opportunities = allPosts.filter(
+    (post) => post.primaryTag.name.toLowerCase() === "opportunities"
+  );
+  return opportunities;
+}
+
+export async function getAllOpportunitiesTags() {
+  const opportunities = await getAllOpportunities();
+  const tags = opportunities.flatMap((post) => post.tags);
+  const uniqueTags = [...new Set(tags.map((tag) => tag.name))];
+  uniqueTags.unshift("All");
+  return uniqueTags;
+}
+
+export async function getOpportnity(slug) {
+  const post = await getPost(slug);
+  return post;
+}
+
+export async function getAllStories() {
+  const allPosts = await getAllPosts();
+  const stories = allPosts.filter(
+    (post) => post.primaryTag?.name.toLowerCase() === "stories"
+  );
+  return stories;
+}
+
+export async function getAllStoriesTags() {
+  const stories = await getAllStories();
+  const tags = stories.flatMap((post) => post.tags);
+  const uniqueTags = [...new Set(tags.map((tag) => tag.name))];
+  uniqueTags.unshift("All");
+  return uniqueTags;
```

Comment:
```suggestion
  return ["All", ...new Set(tags)];
```

Reply:
Perhaps we no longer need the getStoriesTags and getOpportunitesTags since we can just filter out the tags from the list of either stories or opportunities.

Reply:
We need to filter **and** make sure they're unique. I still think a standalone function is much better than duplicating this functionality.